common/src/main/java/com/netflix/conductor/common/metadata/tasks/TaskDef.java

```diff
@@ -242,16 +246,32 @@ public void setRetryDelaySeconds(int retryDelaySeconds) {
 	 *
 	 * @return rateLimitPerSecond The max number of tasks that will be allowed to be executed per second at any given point of time.
 	 */
-	public Integer getRateLimitPerSecond() {
-		return rateLimitPerSecond == null ? 0 : rateLimitPerSecond;
+	public Integer getRateLimitPerFrequency() {
+		return rateLimitPerFrequency == null ? 0 : rateLimitPerFrequency;
+	}
+
+	/**
+	 *
+	 * @param rateLimitPerFrequency The max number of tasks that will be allowed to be executed per second at any given point of time. Setting the value to 0 removes the rate limit
```

Comment:
...number of tasks that will be allowed to be executed per the configured `rateLimitFrequencyInSeconds` ...